godotproject/Level3.gd

```diff
@@ -0,0 +1,41 @@
+extends Node2D
+
+# Each level has to have a name in it now, this will be displayed on the high score screen
+const level_name := "Get to the Rocket!"
```

Comment:
```suggestion
const level_name := "Get to ze Rocket!"
```
Double-check that this matches with what's in the credits screen. I know for all of our reports and stuff we put "ze"

Reply:
Yes, that should be Ze